src/drivers/bitbucket_cloud.js

```diff
@@ -421,6 +425,14 @@ class BitbucketCloud {
     return command;
   }
 
+  get workflowId() {
+    return BITBUCKET_PIPELINE_UUID;
```

Comment:
doesn't this change for each run?

Reply:
Yes. And that's the point - while the user may want to update a comment as the run progresses, they may want to keep comments from previous runs unmodified.

Reply:
I thought the pipeline uuid stayed the same?

Reply:
@casperdcl I think this is safe to resolve, using `BITBUCKET_PIPELINE_UUID` is functionally eq to GH